src/tools/rados/rados.cc

```diff
@@ -302,11 +350,17 @@ static int do_get(IoCtx& io_ctx, RadosStriper& striper,
   int ret;
   while (true) {
     bufferlist outdata;
+
+#ifndef WITH_LIBRADOSSTRIPER
+    ret = io_ctx.read(oid, outdata, op_size, offset);
```

Comment:
@chardan since this is inside the `#else // WITH_LIBRADOSSTRIPER`, the `#ifndef WITH_LIBRADOSSTRIPER` will never be true and the preprocessor will always remove it, right?

Reply:
True, please refer to my  note:
https://github.com/ceph/ceph/pull/18197#issuecomment-335555308

I can remove them if you feel strongly about it (I agree it is gnarly and potentially confusing).

Reply:
I don’t understand this even after reading that note. This is a separate striper-aware compilation path; why would we *add* an implementation that doesn’t contain the striper?